src/lib/components/Selected.svelte

```diff
@@ -8,41 +8,57 @@
 
 	const box = new BoxHelper(new Object3D(), 0x000000)
 	const selected = useSelectedObject()
+	const object3d = $derived.by(() => {
+		if (selected.current === undefined) {
+			return
+		}
+
+		return scene.getObjectByProperty('uuid', selected.current.uuid)
+	})
+	const clone = $derived.by(() => {
+		if (selected.current?.metadata.batched) {
+			return
+		}
+
+		return object3d?.clone(false)
+	})
+
+	const { start, stop } = useTask(
```

Comment:
There was a bug where bounding boxes on the current item would not update in some circumstances. This just makes updates reliable by putting them into the frameloop when something is selected.